tests/cuda/test_spacepoint_formation.cpp

```diff
@@ -25,7 +25,7 @@
 
 using namespace traccc;
 
-TEST(spacepoint_formation, cuda) {
+TEST(CUDASpacepointFormation, cuda) {
```

Comment:
Nit: This is not a very descriptive test name (the "cuda" part) and does not match camelcase naming.